.doc_gen/metadata/iam_metadata.yaml

```diff
@@ -251,6 +251,15 @@ iam_GetAccountPasswordPolicy:
   synopsis: get the &IAM; account password policy.
   category:
   languages:
+    .NET:
+      versions:
+        - sdk_version: 3
+          github: dotnetv3/IAM
+          sdkguide:
+          excerpts:
+            - description: Get the account password policy.
```

Comment:
Only include description if it adds more info than the overall synopsis.

Reply:
removed